docs/gateway.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 5
+---
+
+# Trust Gateway
+
+## Purpose
+
+The Digital Documentation COVID-19 Certificate Gateway (DDCCG) is an extended
+version of the [European Union Digital COVID Certificate Gateway (EUDCCG)](https://ec.europa.eu/health/sites/default/files/ehealth/docs/digital-green-certificates_v2_en.pdf),
+which has enhanced functionality allowing for interoperability between
+multiple gateways in support of the DDCC specification.
+
+While the DDCCG is designed to support the DDCC specification, 
+it also supports various existing digital COVID certificates
+including EUDCC, Smart Health Cards, DIVOC, and ICAO.
```

Comment:
The Trust Gateway specification is based upon the [Digital Documentation COVID-19 Certificate Gateway (DDCCG)](https://github.com/WorldHealthOrganization/ddcc-gateway/blob/main/docs/Architecture.md) and [European Union Digital COVID Certificate Gateway](https://ec.europa.eu/health/sites/default/files/ehealth/docs/digital-green-certificates_v2_en.pdf) (EUDCCG) specification, which were developed as the trust network infrastructure for COVID-19 specific vaccine credentialing. DDCCG is an extended version of EUDCCG, which has enhanced functionality allowing for interoperability between multiple gateways and supports of the DDCC specification. While the DDCCG is designed to support the DDCC specification, it also supports various existing digital COVID certificates including EUDCC, Smart Health Cards, DIVOC, and ICAO.

Reply:
Condensed and updated language for lines 9-16 above^

Attempted to make generic entry to Trust Networks, and identify DDCC and EU specifications as covid examples to reference and base off of